docs/02_installation/README.md

```diff
@@ -88,16 +88,16 @@ Nevertheless, you don't need to be a rocket scientist to install EasyBuild (and
 that's OK too), so don't worry.
 
 You can install EasyBuild just like you can install any other Python software that is released
-via the standard *Python Package Index* (PyPI), through one of standard Python installation tools
-(did you catch the irony in that last bit?). And since EasyBuild is a software installation tool
```

Comment:
Oh :(

Don't you agree the phrase "one of the standard Python installation tools" is ironic?

Reply:
Very! I'm just not happy with the amount of parenthesises around. :-)